addons/vehicles/CfgVehicles/Viper.inc

```diff
@@ -0,0 +1,191 @@
+                class CommanderOptics: CommanderOptics {
+                    weapons[] = {
+                        "HMG_127_APC",
+                        "Laserdesignator_mounted",
+                        "SmokeLauncher"
+                    };
+                    magazines[] = {
+                        "200Rnd_127x99_mag_Tracer_Yellow",
+                        "200Rnd_127x99_mag_Tracer_Yellow",
+                        "200Rnd_127x99_mag_Tracer_Yellow",
+                        "SmokeLauncherMag",
+                        "SmokeLauncherMag",
+                        "SmokeLauncherMag",
+                        "SmokeLauncherMag",
+                        "SmokeLauncherMag",
+                        "Laserbatteries"
+                    };
+                    class Components {
+                        class VehicleSystemsDisplayManagerComponentLeft: VehicleSystemsTemplateLeftCommander {
+                            class Components: components {
+                                class SensorDisplay {
+                                    componentType="SensorsDisplayComponent";
+                                    range[]={4000,2000,1000,8000};
+                                    resource="RscCustomInfoSensors";
+                                };
+                            };
+                        };
+                        class VehicleSystemsDisplayManagerComponentRight: VehicleSystemsTemplateRightCommander {
+                            defaultDisplay="SensorDisplay";
+                            class Components: components {
+                                class SensorDisplay {
+                                    componentType="SensorsDisplayComponent";
+                                    range[]={4000,2000,1000,8000};
+                                    resource="RscCustomInfoSensors";
+                                };
+                            };
+                        };
+                    };
+                };
+            };
+            weapons[] = {
+                "cannon_120mm_long",
+                "LMG_coax"
+            };
+            magazines[] = {
+                "32Rnd_120mm_APFSDS_shells_Tracer_Yellow",
+                "12Rnd_120mm_HEAT_MP_T_Yellow",
+                "12Rnd_120mm_HE_shells_Tracer_Yellow",
+                "4Rnd_120mm_LG_cannon_missiles",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow"
+            };
+        };
+    };
+
+    textureList[] = {
+        "SWS_01", 1,
+        "SWS_02", 1,
+        "SWS_03", 1
+    };
+    class TextureSources {
```

Comment:
It may be wise to extend the base TextureSources so you can use the Indep ones in the garage if you're so inclined.


I'll leave it up to you; since the model will work with the textures I see no reason not to make them available, but it's not necessary.